### PR TITLE
govendor: Fix for Linuxbrew

### PR DESCRIPTION
**Pull request by superbrothers**
This commit fixes not to build for darwin. In this case, GOOS and GOARCH
shoud not be needed to set.

Signed-off-by: Kazuki Suda <kazuki.suda@gmail.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/Linuxbrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Linuxbrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by maxim-belkin** [comment]
add `if OS.mac? to each line instead of removing them

**Comment by superbrothers** [comment]
I fixed.

**Comment by maxim-belkin** [comment]
Fine by me. @sjackman ?

**Comment by superbrothers** [comment]
ping @sjackman

File: Formula/govendor.rb [hunk 1]
```diff
@@ -16,8 +16,8 @@ class Govendor < Formula
 
   def install
     ENV["GOPATH"] = buildpath
-    ENV["GOOS"] = "darwin"
-    ENV["GOARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
+    ENV["GOOS"] = "darwin" if OS.mac?
+    ENV["GOARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386" if OS.mac?
```
Comment (sjackman) [inline Formula/govendor.rb]: Is this line (`GOARCH`) needed?

**Comment by sjackman** [comment]
The bottles expired, sorry. Please rebase onto master.

**Comment by superbrothers** [comment]
I have undone the change of the `GOARCH` line and have rebased.

**Comment by sjackman** [comment]
Merged. Thanks for your contribution to Linuxbrew, Kazuki!

**Pull request closed without merging**